json_text.js

```diff
@@ -1091,29 +1093,29 @@ arrPopups = [
             "<A HREF=\"&#x6d;&#x61;&#x69;&#x6c;&#x74;&#x6f;&#x3a;&#x6c;&#x75;&#x64;&#x6f;&#x6b;&#x40;&#x73;&#x77;&#x69;&#x73;&#x73;&#x74;&#x70;&#x68;&#x2e;&#x63;&#x68;\">" +
             "&#x6c;&#x75;&#x64;&#x6f;&#x6b;&#x40;&#x73;&#x77;&#x69;&#x73;&#x73;&#x74;&#x70;&#x68;&#x2e;&#x63;&#x68;</A>",
          "text_mn": "<div class=\"i-header\">Эх сурвалж</div>" +
-            "Энэхүү тайлбарт зургаар хүрээлэн буй орчны агаарын бохирдлын эрүүл мэндэд үзүүлэх нөлөөллийг харуулсан ба Swiss Tropical ба Нийгмийн эрүүл мэндийн хүрээлэнгийн (Швейцарийн TPH) " +
-            "Швейцарийн уран зохиолын мэдээллийн сан ба Орчны агаарын бохирдлын эрүүл мэндэд үзүүлэх нөлөөллийн үйлчилгээ LUDOK-ийн төслийн хүрээнд боловсруулав. " +
-            "Дараах түнш байгууллагуудтай хамтран боловсрууллаа:" +
+            "Энэхүү интерактив зургаар орчны агаарын бохирдол эрүүл мэндэд үзүүлэх нөлөөллийг харуулав. Swiss Tropical Нийгмийн эрүүл мэндийн хүрээлэнгийн (Швейцарийн TPH) " +
+            "Швейцарийн уран зохиолын мэдээллийн сан, Орчны агаарын бохирдлын эрүүл мэндэд үзүүлэх нөлөөллийн үйлчилгээ LUDOK-ийн төслийн хүрээнд боловсруулав. " +
+            "Дараах түнш байгууллагуудтай хамтран боловсрууллаа. Үүнд:" +
             "<ul>" +
             "<li>Bundesamt für Umwelt</li>" +
             "<li>Kantonale Behörden für Luftreinhaltung</li>" +
             "<li>Krebsliga Schweiz</li>" +
             "<li>LUNGE ZÜRICH</li>" +
             "<li>LerNetz</li>" +
             "</ul>" +
-            "Kонцепц ба хэрэгжилт: LUDOK<br>" +
+            "Kонцепц хөгжүүлэлт: LUDOK<br>" +
             "Зохиогчид: Meltem Kutlar Joss, Lara Stucki, Ron Kappeler, Zoe Roth<br>" +
-            "Визуал дизайн ба програмчлал: <a href=\"https://www.dr-carl.com\" target=\"_blank\">Dr. Carl GmbH</a>" +
+            "Визуал дизайн, програмчлал: <a href=\"https://www.dr-carl.com\" target=\"_blank\">Dr. Carl GmbH</a>" +

```

Comment:
I would have expected to see names of translators for the Mongolian version or at least the name of your organization.